src/lib.rs

```diff
@@ -13,14 +17,13 @@ use http_body_util::{BodyExt, Full};
 use hyper::body::{Bytes, Incoming};
 use hyper::header::{CONTENT_TYPE, LOCATION};
 use hyper::{Method, Request, Response, StatusCode};
+use hyper_util::client::legacy::{connect::Connect, Client as HyperClient};
```

Comment:
I prefer to stick to one line per module (so keeping the separate lines for `Connect` and `Client`, as before).